build.assets/Dockerfile

```diff
@@ -114,7 +114,8 @@ ARG BUILDARCH
 
 # Install packages.
 # Java JRE is required by gcloud firestore emulator.
-# Latest git 2.18+ is required for GitHub actions
+# Latest git 2.18+ is required for GitHub actions.
+# rsync is required for some integration tests.
```

Comment:
Can you add this comment inline above the rsync itself? Similar to that we've done here: https://github.com/gravitational/teleport/blob/master/build.assets/Dockerfile-centos7#L178-L200
It's easier to trace it later.

Reply:
Sure, done